src/scripts/config/mlz_config_create.sh

```diff
@@ -70,12 +70,52 @@ if [[ -z $(az ad sp list --filter "displayName eq '${mlz_sp_name}'" --query "[].
         --output tsv)
 
     # Get Service Principal AppId
+    # Added the sleep below to accomodate for the transient behavior where the Service Principal creation
```

Comment:
For both of these while loops that attempt for a given maximum time/attempts, let's follow [the pattern we use in configure_asc.sh](https://github.com/Azure/missionlz/blob/10501eb2a6ff605aa17eb2cbd6ce56c9195b0e0a/src/scripts/security-center/configure_asc.sh#L141-L155) that notifies the user that work is occurring, how long they'll be waiting, and how many attempts are left before giving up and exiting: https://github.com/Azure/missionlz/blob/10501eb2a6ff605aa17eb2cbd6ce56c9195b0e0a/src/scripts/security-center/configure_asc.sh#L141-L155

Reply:
Updated logic to follow pattern